vector-im/element-android
# Voice broadcast - live listening

## Type of change


## Content

Be able to listen to an ongoing voice broadcast
When a voice broadcast is live, trigger the play action will start listening to the last received chunk and wait for the new ones with a "buffering" state in the meantime.
Also, when the recording is paused, the current chunk is now stopped and sent to the room. Resume the recording will start a new chunk file.

Note: the UI will be improved in a dedicated PR for a better rendering of the listening state

## Motivation and context

Continue #7127

## Screenshots / GIFs

## Tests

- Start playing a live voice broadcast
- Verify that we are able to listen it until it is stopped or paused by the recorder
- Verify that when the recorder resume the voice broadcast, you continue to listen to the new chunks
- Verify that the listener is able to pause and resume the listening without problem

## Tested devices

- OS version(s):

## Checklist
